test/cabbage_domain.pddl

```diff
@@ -0,0 +1,42 @@
+; implementation of the https://en.wikipedia.org/wiki/Wolf,_goat_and_cabbage_problem
```

Comment:
Can these `.pddl` files just live in the test directory, or should we move them elsewhere.